### ISSUE TITLE
♿Slider AVT1 - August 2018♿

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1364]
## Environment
Chrome Version 70.0.3538.67 (Official Build) (64-bit)
macOS Mojave 10/Windows 10

## Violations:
Each form control must have associated label.
<img width="410" alt="screen shot 2018-10-30 at 2 03 20 am" src="https://user-images.githubusercontent.com/40970507/47701525-166df280-dbe8-11e8-85f9-cf0fa9f98d5c.png">

The id slider referenced by the 'for' attribute of a label element is not a valid form input element or an element with id slider is either null or does not exist.
<img width="414" alt="screen shot 2018-10-30 at 2 04 35 am" src="https://user-images.githubusercontent.com/40970507/47701550-2b4a8600-dbe8-11e8-9060-707121cae1d4.png">

## How to reproduce
1. [Download the Dynamic Accessibility Plugin tool here](https://ibm.ent.box.com/notes/335005856253?s=mhpgjlgss354mcidsv6rvpuu0bxlmfm4)
2. Run the extension using the August 2018 ruleset
